### PR TITLE
Add marker sources

### PR DESCRIPTION
**Pull request by vincerubinetti**
- add markers/overlays/locations from cancerinfocus.org. taken from the *us_facilities_and_providers* csv file. missing coordinates filled in from Google Maps API based on address (if present).
- add markers from [CU Cancer Center](https://medschool.cuanschutz.edu/colorado-cancer-center/community/CommunityOutreachEngagement/Colorado-Cancer-map) ([full screen map link](https://ucdamc-geo.maps.arcgis.com/apps/MapSeries/index.html?appid=9201eee6964541d7bc56539e2685177f)), which appears to get its data via [this request](https://ucdamc-geo.maps.arcgis.com/sharing/rest/content/items/5aaf9f6df8584a5aa6828ba118527e2b/data?f=json). missing coordinates filled in from Google Maps API based on address (if present). no "human readable" labels for the categories appear to exist here. for example, i assume `CCSP_15Jun2021` refers to the "cancer center screening program" tab on the CU page, but i can't find a match for `WWC_15Jun2021` for example. 
- update frontend packages to latest
- refactor util formatting/percent funcs a little
- tweak some styles
- tweak presentation of tooltip

### CODE REVIEW

**Comment by vincerubinetti** [comment]
@janolowery @BORRAYOE I could use your help regarding the label issue mentioned above.

**Review by falquaddoomi — APPROVED** [review]
Looks good! Nice work on the marker legend and popups for each of those, and for marshalling all that data.

File: docker-compose.override.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ services:
 
   frontend:
     ports:
-      - 8021:8001
+      - 8022:8001
```
[the remark this answers is not in the document]
Comment (vincerubinetti) [reply docker-compose.override.yml:12]: Oh yeah sorry I forgot to change that back, something was running on that but couldn't find what it was.

[the remark this answers is not in the document]
Comment (falquaddoomi) [reply docker-compose.override.yml:12]: No worries, it's fine to leave it at that if it works for you; I chose the original ports arbitrarily thinking they would be unused.

[the remark this answers is not in the document]
Comment (vincerubinetti) [reply docker-compose.override.yml:12]: I don't think either of them are used by anything I typically use (like Vite). I just haven't restarted my laptop in many months and have done a bunch of whacky stuff, so there probably won't be a collision again. I'll leave it as is out of laziness.

**Pull request merged**